packages/docs/content/components/step-list.mdx

```diff
@@ -35,13 +20,23 @@ The `StepList` component is the preferred method for building a step list, since
   storyId="patterns-step-list--step-list-example"
 />
 
-### Props
+## Code
+
+### `<StepList>` component
```

Comment:
I think this needs to be updated to `### React` and then `#### Steplist` like the other pages - see Checkbox for an example

Reply:
Good catch 👍